subjects/index.js

```diff
@@ -0,0 +1,40 @@
+const pg = require('pg');
+
+function executeSQL(context, params) {
+    const connectionString = 'postgres://postgres:postgres@localhost:5432/sample_db'
+    const client = new pg.Client({ connectionString })
+
+    client.connect();
+    const { maxLat, minLat, maxLon, minLon } = params
+
+    if (!maxLat || !minLat || !maxLon || !minLon) {

```

Comment:
Optional: you could throw this `if` clause into its own function (e.g. `validateParams`) to shrink the size of this main `executeSQL` function. I dont have a strong opinion on this, and it's not like `executeSQL` function is painfully long right now, its just a matter of stylistic preference. If we add any additional validation here though, I would definitely recommend extracting it into its own function.

Reply:
I'll leave this one as is for now, since these are likely the only two functions we'll have for the time being.